lib/fold-comments.coffee

```diff
@@ -27,8 +27,11 @@ module.exports = FoldComments =
     editor ||= atom.workspace.getActiveTextEditor()
 
     eachFoldable = (f) ->
+      lastFoldedRow = -2
```

Comment:
`-1` would cause row 0 to be ignored. Could be any value less than `-1`, or an `INVALID` kind of identifier if coffeescript provides that.